.github/workflows/release-subsquid-processor.yml

```diff
@@ -0,0 +1,64 @@
+name: "Release Subsquid Production Image"
+env:
+  SUBSQUID_IMAGE_RELESE_TAGS: ${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:latest,${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:{{ github.event.pull_request.head.sha }},v0.0.1
```

Comment:
type on RELESE

Reply:
Also, v0.0.1 is hardcoded?

Reply:
Also, currently build-test-deploy will upload the same image under the same tags I think

Reply:
fixed this one :)